src/posts/PostList.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import Post from './Post';
+import { useFilterStore } from '../helpers/filter-store';
+import "./PostList.scss"
+
+const PostList = () => {
+  const { verified } = useFilterStore();
+  const [posts, setPosts] = useState([] as Array<{ is_verified: boolean }>);
```

Comment:
this state should use `Post` type that you will create
`const [...] = useState<Post[]>([]); `